### PR TITLE
Nalimilan/destdir

### PR DESCRIPTION
**Pull request by nalimilan**
With this I'm able to build the RPM package fine.

### CODE REVIEW

**Comment by nalimilan** [comment]
Regarding the symlinks, I wasn't sure whether creating them would make any sense on OSX.

File: Makefile [hunk 1]
```diff
@@ -74,12 +75,16 @@ libutf8proc.dylib: libutf8proc.$(MAJOR).dylib
...
-	$(INSTALL) -m 755 libutf8proc.$(SHLIB_VERS_EXT) $(libdir)
-	ln -f -s $(libdir)/libutf8proc.$(SHLIB_VERS_EXT) $(libdir)/libutf8proc.$(SHLIB_EXT)
+	mkdir -m 755 -p $(DESTDIR)$(includedir)
+	$(INSTALL) -m 644 utf8proc.h $(DESTDIR)$(includedir)
+	mkdir -m 755 -p $(DESTDIR)$(libdir)
+	$(INSTALL) -m 644 libutf8proc.a $(DESTDIR)$(libdir)
+	$(INSTALL) -m 755 libutf8proc.$(SHLIB_VERS_EXT) $(DESTDIR)$(libdir)
+	ln -f -s $(libdir)/libutf8proc.$(SHLIB_VERS_EXT) $(DESTDIR)$(libdir)/libutf8proc.$(SHLIB_EXT)
+	ifneq ($(OS),Darwin)
+	ln -f -s $(libdir)/libutf8proc.so.$(MAJOR).$(MINOR).$(PATCH) $(DESTDIR)$(libdir)/libutf8proc.so.$(MAJOR)
```
Comment (stevengj) [inline Makefile]: Probably better to do `ln -f -s $(libdir)/libutf8proc.$(SHLIB_VERS_EXT) ....` since we have this var, for consistency.

Comment (nalimilan) [inline Makefile]: Right, I've fixed it.

File: Makefile [hunk 2]
```diff
@@ -28,6 +28,7 @@ else # GNU/Linux, at least (Windows should probably use cmake)
 endif
 
 # installation directories (for 'make install')
+DESTDIR=
```
Comment (stevengj) [inline Makefile]: is this line needed? see also @tkelman's comment

Comment (stevengj) [inline Makefile]: I guess it is harmless.

Comment (nalimilan) [inline Makefile]: Ah, I just removed it...

**Pull request merged**